src/annotations/components/EditAnnotationForm.tsx

```diff
@@ -63,8 +67,13 @@ export const EditAnnotationForm: FC<EditAnnotationProps> = ({
     }))
   }
 
-  const annotationStreams = useSelector(getAnnotationStreams)
+  const handleDeleteAnnotation = () => {
+    dispatch(deleteAnnotations(editAnnotationState))
```

Comment:
can you add two metric calls here (one for success, and one for failure, when you move the failure calls up to this layer), and then connect this PR to https://github.com/influxdata/ui/issues/728 and check the one check box related to deleting annotations please?